plugins/uefi/fu-plugin-uefi.c

```diff
@@ -29,6 +29,10 @@
 #include "fu-plugin.h"
 #include "fu-plugin-vfuncs.h"
 
+struct FuPluginData {
+	const gchar	*ux_capsule;
```

Comment:
I'd rather this was a gboolean, and then we can marshal it to a string when required.

Reply:
Adjusted